lib/skate_web/router.ex

```diff
@@ -40,6 +40,13 @@ defmodule SkateWeb.Router do
     plug :put_secure_browser_headers
   end
 
+  scope "/docs" do
+    for {key, value} <- Application.compile_env(:skate, SkateWeb.ConfigPagesController)[:pages] do
+      get key, SkateWeb.ConfigPagesController, :index,
+        private: %{SkateWeb.ConfigPagesController => %{external: value}}
+    end
+  end
```

Comment:
RFC: I could try pulling this out into a Plug, so that we don't need to do the `for` loop directly within `router.ex`.